packages/input-select/src/Select.story.js

```diff
@@ -95,4 +95,18 @@ storiesOf("Select Input", "module")
         />
       </Wrapper>
     ))
+  )
+  .add(
+    "Read Only",
+    withInfo()(() => (
+      <Wrapper>
+        <Select
+          placeholder="Select fruit"
+          label="Fruit"
+          items={items}
+          onChange={action("onChange")}
+          readOnly
```

Comment:
`readOnly` gives me the idea that the Select is disabled. The goal of this property is to disable the search feature.

Our suggestion is to receive a property called `disableAutoComplete` or something like that and disable the formInput based on this property.

Reply:
Or even `disableSearch`